lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -63,23 +73,37 @@ def withdrawn?
     end
 
     def expanded_links
-      payload["expanded_links"] || {}
+      @_expanded_links ||= payload["expanded_links"] || {}
+    end
+
+    def expanded_links_item(key)
+      expanded_links[key] || {}
+    end
+
+    def values_from_collection(collection, key)
+      collection.map { |item| item[key] }
     end
 
-    def organisations
-      expanded_links["organisations"] || {}
+    def organisation_slugs(type)
+      expanded_links_item(type).map do |content_item|
+        content_item["base_path"].sub('/government/organisations/', '').sub('/courts-tribunals/', '')
```

Comment:
Yup, this is rummager weirdness IMO. Really there's no need to have it without the leading slash but we're leaving it like that to keep API compatibility.

If it's cleaner we could also move this to the presentation layer: https://github.com/alphagov/rummager/blob/master/lib/search/presenters/result_presenter.rb